Libraries/WebSocket/__tests__/Websocket-test.js

```diff
@@ -0,0 +1,207 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+'use strict';
+
+jest
+  .autoMockOff()
```

Comment:
@cpojer I wish I could, but I couldn't make that work (ran into all kinds of errors, even when I was dontMock'ing all the things I needed). If you want to have a try, go ahead :)